packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -0,0 +1,128 @@
+import * as core from '@aws-cdk/core';
+import * as cfn_type_to_l1_mapping from './cfn-type-to-l1-mapping';
```

Comment:
Our naming convention is camel case. let's use that.

Reply:
Really? That's news to me... We use `codepipeline_actions` in the documentation and examples for `@aws-cdk/aws-codepipeline-actions`, and `cw_actions` for `@aws-cdk/aws-cloudwatch-actions`...

I do see camelCase in `@aws-cdk/alexa-ask` (which is CFN-only) and `@aws-cdk/region-info`, but I'm not sure it's obvious that's the convention...

Reply:
I think the point of using snake_case is to clearly distinguish namespace qualifiers from class names. Since classes also use camelCase, this makes using them with camelCase import qualifiers less readable (for example, I would argue `codepipeline_actions.SomeAction` is more readable than `codepipelineActions.SomeAction`).